src/tokens/functional/size/size.json

```
@@ -51,7 +52,8 @@
         "$type": "dimension"
       },
       "paddingBlock": {
-        "value": "calc(({control.small.size} - {control.small.lineBoxHeight}) / 2)"
+        "$value": "4px",
```

```suggestion
        "$value": "{base.size.4}",
```

Might as well use the base token unless you think its confusing to mix with the others using pixel